spring-integration-gcp/pom.xml

```diff
@@ -21,10 +21,18 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-file</artifactId>
```

Comment:
`optional`.
Someone might not be interested in Google Cloud Storage, but just PubSub.

Similar situation we have with the SI-AWS and I don't think that it is going to be good to granulate to separate short modules. End user will be forced to add an extra dependency anyway

Reply:
Why don't you think it would be good to have separate sub-modules like `spring-integration-pubsub` and `spring-integration-storage`? The `spring-integration` project is modular itself (e.g., `spring-integration-amqp`, `spring-integration-jms`, etc.)
Asking because I meant to split the `spring-integration-gcp` module into two after this PR.

Reply:
Right, it is but, pay attention how all the protocol related stuff is collected under the single module, like AWS or JMX or XML. In thins case it is GCP protocol, so I tend to see all the Spring Integration GCP stuff in a single module.

Any other opinions?

Reply:
OK cool, I'll keep everything under the same module for now!